src/services/messagesService.js

```diff
@@ -178,6 +178,12 @@ const getReactionsDetails = async function(token, messageId) {
 	return axios.get(generateOcsUrl('apps/spreed/api/v1/reaction/{token}/{messageId}', { token, messageId }))
 }
 
+const editMessage = async function({ token, messageId, updatedMessage }, options) {
```

Comment:
This will create confusion at some point because editing messages touches both the original message and the new one so it's better to indicate which one is being passed (although it may seem obvious for now)